docs/source/zh/_toctree.yml

```
@@ -50,6 +50,8 @@
     title: 导出为 TFLite
   - local: torchscript
     title: 导出为 TorchScript
+  - local: gguf
+    title: 与 GGUF 格式共享模型
```

```suggestion
    title: 与 GGUF 格式的互操作性
```
The "Interoperability" should be translated to "互操作性" correctly, since it's a computer terminology.